README.md

```diff
@@ -104,7 +104,12 @@ bundle exec rake test:extensions
 bundle exec rake test:all
 ```
 
-It is recommended to run the last command at least once right before pushing.
+### Run absolutely all the tests plus rubocop and bundle audit
```

Comment:
I'm a little concerned that this line will drift out of sync as we add other default tasks, but I can live with that for now.